api/src/main/java/org/openmrs/module/pihcore/task/PihCloseStaleVisitsTask.java

```diff
@@ -42,89 +43,108 @@ public class PihCloseStaleVisitsTask implements Runnable {
 
     private static final String ENCOUNTER_TYPE_ED_TRIAGE_UUID = PihEmrConfigConstants.ENCOUNTERTYPE_EMERGENCY_TRIAGE_UUID;
 
+    private static boolean isExecuting = false;
+
     @Override
     public void run() {
```

Comment:
I can move all of this to a separate PR if desired.  Most of these changes are just whitespace.  I just added some additional logging and a check to ensure this doesn't run if it is already running.

Reply:
to clarify my previous comment, don't worry about moving to a separate PR unless you want to get this in faster.